src/app/product/Product.module.scss

```
@@ -0,0 +1,236 @@
+@import '../variables.scss';
+
+.container {
+    min-height: 50vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    padding: 44px 0;
+    @include responsive-padding;
+}
+
+.card {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    @media (max-width: 950px) {
+        flex-direction: column;
+    }
+}
+
+.card > * {
+    width: 50%;
+    @media (max-width: 950px) {
+        width: 100%;
+    }
+}
+
+.image img {
+    width: 100%;
+    height: auto;
+    object-fit: contain;
+}
+
+.info {
+    padding-top: 40px;
+}
+
+.isAvailable {
+    color: #1FD85D;
+    font-weight: 500;
+}
+
+.title, .title * {
+    font-size: 30px;
+    font-weight: 400;
+    margin: 10px 0 10px 0;
+    line-height: 1.5;
+}
+
+.title .titleHighlight {
+    font-weight: 800;
+}
+
+.weight {
+    display: flex;
+    align-items: center;
+    align-content: center;
+}
+
+.weight img {
+    margin-right: 5px;
+}
+
+.weight span {
+    color: #4e4e4e;
+}
+
+.PriceCartContainer {
+    display: flex;
+    justify-content: start;
+    align-items: center;
+    margin-top: 20px;
+    @media (max-width: 950px) {
+        flex-wrap: wrap;
+    }
+}
+
+.buttonContainer {
+    @media (max-width: 950px) {
+        min-width: fit-content;
+        width: 50%;
+    }
+    @media (max-width: 711px) {
+        margin-top: 16px;
+    }
+}
+
+.PriceQuantity {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: nowrap;
+}
+
+.price {
+    font-size: 30px;
+    font-weight: 800;
+    white-space: nowrap;
+}
+
+.quantity {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 30px;
+}
+
+.quantity > * {
+    margin: 0 10px;
+}
+
+.minusplus {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 30px;
+    height: 20px;
+    background: #ffefd1;
+    border-radius: 10px;
+    cursor: pointer;
+    transition: background 0.3s ease;
+}
+
+.minusplus:hover {
+    background: #ffe494;
+}
+
+.outlines {
+    display: flex;
+    justify-content: space-between;
+    margin: 40px 0;
+
+    @media (max-width: 711px) {
+        flex-direction: column;
+    }
+}
+
+.outlines > * {
+    border-radius: 12px;
+    height: 88px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 28px;
+    margin: 0 5px;
+    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.04);
+    color: #3f4e65;
+
+    @media (max-width: 711px) {
+        margin: 5px 0;
+    }
+}
+
+.PriceList {
```

```suggestion
.priceList {
```
Лучше везде с маленькой буквы писать.